src/mmw/js/src/modeling/templates/projectMenu.html

```diff
@@ -7,7 +7,7 @@
             {% endif %}
         </button>
         {% if editable or is_new or itsi %}
-            <ul class="dropdown-menu menu-right" role="menu">
+            <ul class="dropdown-menu menu-left" role="menu">
```

Comment:
![screen shot 2017-06-14 at 11 35 36 am](https://user-images.githubusercontent.com/7633670/27153821-e6c5888e-5120-11e7-857e-e237e58d0974.png)

The dropdown was getting cutoff with shorter project titles.